NEWS.md

```diff
@@ -20,6 +20,21 @@ For example:
 > - [bytecode_api.h](headers/bytecode_api.h)
 > - [bytecode_local.h](headers/bytecode_local.h)
 
+## `1.3.0-rc`
+
+➕ Upgrade bytecode compiler project to LLVM 16.
+  - The bytecode compiler project now builds multiple shared object files,
+    instead of just one with all of the passes.  This is due to running with
+    the "new" pass manager, instead of running with the legacy pass manager,
+    as before.
```

Comment:
It would be nice to add reference links - one or both of these maybe:
- https://llvm.org/docs/NewPassManager.html 
- https://blog.llvm.org/posts/2021-03-26-the-new-pass-manager/

Reply:
Agree, done.